src/libraries/Microsoft.Extensions.Configuration.Binder/tests/SourceGenerationTests/GeneratorTests.Baselines.cs

```diff
@@ -851,7 +852,7 @@ public async Task CollectionsFwk()
         {
             string source = GetCollectionsSource();
 
-            await VerifyAgainstBaselineUsingFile("Collections.net462.generated.txt", source, assessDiagnostics: (d) =>
```

Comment:
Per https://github.com/dotnet/runtime/pull/89952/commits/f5a2abcfc0dfc5fdb0267c378dd2be745fcda39b#r1321885499, please revert the changes to these invocations.